ansible/roles/zsh/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+- name: Install zsh
```

Comment:
Since we're going to have a bunch of yaml stuff here, I'd like to see a yaml linter added as well (and maybe autoformatter?)

Reply:
I can enable prettier for yaml if I haven't already

Reply:
There's also `yamllint` that I've used in other projects

Reply:
We do have a linter for yml already enabled.